Headers/Date.h

```diff
@@ -0,0 +1,28 @@
+#ifndef __DATE_H__
+#define __DATE_H__
+
+
+/**
+* @class Date
+* class for saving date by 3 unsigned numbers;
+* input, print methods to work with class in console;
+* methods fin/fout takes data from/to file;
+* can be constructed empty or with 3 unsigned in arguments (day, month, year);
+*/
+class Date {
+
+    public:
+        unsigned day, month, year;
+
+        void input();
+        void print();
+        int fin(const char* fname);

```

Comment:
/*
   Input date from file
  @arg fname - const char* - name of th input file
 @return 0 in the case of success, number of error otherwise
**/
         int fin(const char* fname); /

Reply:
1. Look at my last comments
2 . Your choice - only document it somewhere
3. Предполагается несколько файлов - 1.Данные по работникам, 2. Данные по оборудованию, 3. Связи .
Можете сделать свой вариант, лишь бы работало
4.  Your choice